### ISSUE TITLE
Auro-Icon: [IMPORTANT!] statusWarning color fails a11y check

### ISSUE THREAD

**Issue by rallyb13** [AlaskaAirlines/auro-icon#201]
### What browser client(s) are you seeing the issue on?

- [ ] Firefox
- [x] Chrome
- [ ] Safari
- [ ] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [x] Desktop
- [x] Mobile

### Steps to reproduce

1. Anywhere with a standard/white backround, use auro-icon with variant="statusWarning"
2. Add text by setting label to true and setting text within icon tags
3. See bright yellow text on background. The more text you have, the harder it gets to read.
<img width="1026" height="433" alt="Image" src="https://github.com/user-attachments/assets/8098fa15-de3f-4e77-a1f5-95b42edadb35" />
4. (optional) Run Chrome's Lighthouse Accessibility check. It fails contrast ratio check for text on the page.
<img width="620" height="533" alt="Image" src="https://github.com/user-attachments/assets/16112686-a786-4fa3-ab8a-9aae9d72903d" />

### Expected behavior

Contrast ratio is at least 4.5 for text (but we don't need to stick to the minimum).

### Which version of auro-templates do you have installed?

8.1.2 of auro-icon

### Reproducible on the Auro doc site

- [x] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by megrenn** [comment]
solved for an immediate solution for the reporter - Forrest
